### ISSUE TITLE
applying replacements to #[doc = r"..."] fails

### ISSUE THREAD

**Issue by drahnr** [drahnr/cargo-spellcheck#18]
```diff
diff --git a/demo/src/lib.rs b/demo/src/lib.rs
index ffa9b4a..598da32 100644
--- a/demo/src/lib.rs
+++ b/demo/src/lib.rs
@@ -16,7 +16,7 @@ pub fn c() {
 
 }
 
-#[doc = r#"Risk is not your thing, just tkae the money and run."#]
+#[doc = r#"Risk is not your thing, just tktakehe money and run."#]
 pub fn take_the_money_and_leave() {
 
-}
\ No newline at end of file
+}
```

**Comment by drahnr** [comment]
Cc @KuabeM

**Comment by drahnr** [comment]
Resolved by merge of #55
